frontend/src/relay/errors.ts

```diff
@@ -0,0 +1,33 @@
+import {
+    GraphQLSingularResponse,
+    GraphQLResponseWithData,
+    GraphQLResponseWithoutData,
+} from "relay-runtime";
+
+
+export class ServerError extends Error {
```

Comment:
Maybe add comments to describe the purpose of `ServerError` and `APIError` below?